src/components/landing-hero/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { useCurrentProduct } from 'contexts'
+import IconTileLogo from 'components/icon-tile-logo'
+import Heading from 'components/heading'
+import Text from 'components/text'
+import { LandingHeroProps } from './types'
+import s from './landing-hero.module.css'
+
+const LandingHero = ({ pageHeading, pageSubtitle }: LandingHeroProps) => {
```

Comment:
This, and the associated CSS, hasn't been changed at all relative to #733. Presentation layer FTW 🙌